corehq/apps/app_manager/app_translations/parser.py

```diff
@@ -74,8 +74,10 @@ def _get_rows_for_module_sheet(consolidated_po_entries):
                 index = int(occurrence[1])
                 rows[index] = po_entry
         # ensure the number of final translations is same as the highest index
-        if rows:
-            assert len(rows) == int(max(rows.keys()))
+        # if rows:
+            # ToDo: Add this message
+            # if len(rows) != int(max(rows.keys())):
+                # add a message for the user
```

Comment:
Should this commented out code be removed?

Reply:
would cover this in a follow up PR as an improvement, not a blocker for the fix being done here.